docs/declarative-customization/site-theming/sharepoint-site-theming-powershell.md

```diff
@@ -141,40 +142,43 @@ Custom Cyan
 ```
 ## Remove-SPOTheme
 
-This command removes a theme from your tenant store. For example, this command removes the "Custom Cyan" theme that was used in the above examples:
+The **Remove-SPOTheme** command removes a theme from your tenant store. For example, this command removes the "Custom Cyan" theme that was used in the previous examples.
+
 ```powershell
 c:\> Remove-SPOTheme -Name "Custom Cyan"
 ```
 ## Set-HideDefaultThemes
 
-_NOTE: this command will be renamed to ```Set-SPOHideDefaultThemes``` in a future release, for consistency with other SharePoint PowerShell commands. This document will be updated to reflect the change at that time._
+_NOTE: this command will be renamed to ```Set-SPOHideDefaultThemes``` in a future release, for consistency with other SharePoint PowerShell commands._
```

Comment:
Let's use the >[!NOTE] syntax supported on OPS. It will format these correctly.

Reply:
Fixed.